redaxo/src/addons/debug/lib/sql_debug.php

```diff
@@ -1,26 +1,26 @@
 <?php
 
-rex_extension::register('OUTPUT_FILTER', ['rex_sql_debug', 'doLog']);
-
 /**
- * Class to monitor sql queries.
- *
- * @author staabm
- *
  * @package redaxo\debug
+ *
+ * @internal
  */
 class rex_sql_debug extends rex_sql
 {
-    private static $queries = [];
-    private static $errors = 0;
-
     /**
      * {@inheritdoc}.
      */
     public function setQuery($qry, array $params = [], array $options = [])
     {
         try {
+            $timer = new rex_timer();
             parent::setQuery($qry, $params, $options);
+
+            // to prevent double entries, log only if no params are passed
```

Comment:
D.h. Queries mir params werden gar nicht kommen, auch nicht wenn sie nur 1x ausgeführt wurden?

Reply:
Nein, die Queries mit Params gehen von setQuery weiter in execute(), und die methode wird weiter unten separat abgefangen und an clockwork übertragen. Deswegen werden die Fälle hier ignoriert.